tests/e2e/csi_snapshot_basic.go

```diff
@@ -4918,4 +4930,145 @@ var _ = ginkgo.Describe("[block-vanilla-snapshot] Volume Snapshot Basic Test", f
 		err = verifySnapshotIsDeletedInCNS(volHandle, snapshotId)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	})
+
+	/*
+		Scale-up creation of snapshots across multiple volumes
+
+		1. Create a few pvcs (around 25)
+		2. Trigger parallel snapshot create calls on all pvcs
+		3. Trigger parallel snapshot delete calls on all pvcs
+		4. All calls in (2) and (3) should succeed since these are
+		   triggered via k8s API (might take longer time)
+		5. Trigger create/delete calls and ensure there are no stale entries left behind
+		6. Create multiple volumes from the same snapshot
+	*/
+
+	/*
+	   Scale up the total number of snapshots in a cluster, by increasing the volume counts
+
+	   1. Create several 100 of pvcs  (say 500)
+	   2. Maximize the snapshots on each volumes by reaching max_snapshots_per_volume on each volume (create around 2000 snapshots)
+	   3. At this scale, try a few workflow operations such as below:
+	   4. Volume restore
+	   5. snapshot create/delete workflow
+	*/
+	ginkgo.It("Scale-up creation of snapshots across multiple volumes", func() {
```

Comment:
can you add flavours on which this test case needs to be added like examples [csi-block-vanilla][csi-block-vanilla-parallelized]

Reply:
The tests in the whole class file belong to vanilla - and it is declared in the class level

[block-vanilla-snapshot]

https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/1874/files#diff-162ae87d5572d3464a4d07cddec2be887380feee398c148fbe1d26e040ba0fcaR49